src/common/reconciler.go

```diff
@@ -229,58 +202,83 @@ func (r *ServiceReconciler) updateService(service *opslevel.Service, registratio
+	if registration.Description != "" && registration.Description != service.Description {
+		updateServiceInput.Description = opslevel.RefOf(registration.Description)
+	}
+	if registration.Framework != "" && registration.Framework != service.Framework {
+		updateServiceInput.Framework = opslevel.RefOf(registration.Framework)
+	}
+	if registration.Language != "" && registration.Language != service.Language {
+		updateServiceInput.Language = opslevel.RefOf(registration.Language)
+	}
+	if registration.Lifecycle != "" && registration.Lifecycle != service.Lifecycle.Alias {
+		if lifecycle, ok := opslevel.Cache.TryGetLifecycle(registration.Lifecycle); ok {
+			if lifecycle == nil {
+				err := fmt.Errorf("the cache unexpectedly returned a lifecycle that is nil - please submit a bug report")
+				log.Warn().Msgf("[%s] unexpected happened: %v", service.Name, err)
+			} else {
+				updateServiceInput.LifecycleAlias = opslevel.RefOf(lifecycle.Alias)
+			}
+		} else if registration.Lifecycle != "" {
+			log.Warn().Msgf("[%s] Unable to find 'Lifecycle' with alias '%s'", service.Name, registration.Lifecycle)
+		}
 	}
-	if registration.System != "" {
-		updateServiceInput.Parent = opslevel.NewIdentifier(registration.System)
+	if registration.Name != "" && registration.Name != service.Name {
+		updateServiceInput.Name = opslevel.RefOf(registration.Name)
 	}
-	if v, ok := opslevel.Cache.TryGetTier(registration.Tier); ok {
-		if v == nil {
-			err := fmt.Errorf("the cache unexpectedly returned a tier that is nil - please submit a bug report")
-			log.Warn().Msgf("[%s] unexpected happened: %v", service.Name, err)
-		} else {
-			updateServiceInput.TierAlias = opslevel.RefOf(v.Alias)
+	if registration.Owner != "" && registration.Owner != service.Owner.Alias {
+		if team, ok := opslevel.Cache.TryGetTeam(registration.Owner); ok {
+			if team == nil {
+				err := fmt.Errorf("the cache unexpectedly returned a team that is nil - please submit a bug report")
+				log.Warn().Msgf("[%s] unexpected happened: %v", service.Name, err)
+			} else {
+				updateServiceInput.OwnerInput = opslevel.NewIdentifier(team.Alias)
+			}
+		} else if registration.Owner != "" {
+			log.Warn().Msgf("[%s] Unable to find 'Team' with alias '%s'", service.Name, registration.Owner)
 		}
-	} else if registration.Tier != "" {
-		log.Warn().Msgf("[%s] Unable to find 'Tier' with alias '%s'", service.Name, registration.Tier)
 	}
-	if v, ok := opslevel.Cache.TryGetLifecycle(registration.Lifecycle); ok {
-		if v == nil {
-			err := fmt.Errorf("the cache unexpectedly returned a lifecycle that is nil - please submit a bug report")
-			log.Warn().Msgf("[%s] unexpected happened: %v", service.Name, err)
-		} else {
-			updateServiceInput.LifecycleAlias = opslevel.RefOf(v.Alias)
-		}
-	} else if registration.Lifecycle != "" {
-		log.Warn().Msgf("[%s] Unable to find 'Lifecycle' with alias '%s'", service.Name, registration.Lifecycle)
+	// TODO: alias equivalency - need to check ALL the aliases
```

Comment:
might be worth being a little more descriptive here incase we don't get back around to it quickly.  Call out specifically that we should be checking a cache of system aliases.